server/adaptors/integrations/repository/integration.ts

```diff
@@ -211,7 +211,7 @@ export class Integration {
       );
       try {
         const ndjson = await fs.readFile(sobjPath, { encoding: 'utf-8' });
-        const asJson = '[' + ndjson.replace(/\n/g, ',') + ']';
+        const asJson = '[' + ndjson.trim().replace(/\n/g, ',') + ']';
```

Comment:
Should probably include a test case that has an empty line to avoid regressions if we refactor this later.

Reply:
why should we test this case ? its not transitioning the data but only removed none necessary empty lines - if none exist it does nothing...
how would we possibly break this in the future ?

Reply:
If it's simple to test, "how would we possibly break this in the future" is exactly the kind of unknown to test-proof.  since this is a data-modifying function, I'd lean toward producing some sort of test.